src/test/skript/tests/syntaxes/expressions/ExprClamp.sk

```diff
@@ -0,0 +1,119 @@
+test "clamp expression":
+	# Upper and lower limits
+
+	# Normal Cases
+	assert 1 clamped between 0 and 2 is 1 with error "(single ints) min < value < max", expected 1, got 1 clamped between 0 and 2
+	assert 1 clamped between 1 and 2 is 1 with error "(single ints) min = value < max", expected 1, got 1 clamped between 1 and 2
+	assert 2 clamped between 1 and 2 is 2 with error "(single ints) min < value = max", expected 2, got 2 clamped between 1 and 2
+	assert 0 clamped between 1 and 2 is 1 with error "(single ints) value < min < max", expected 1, got 0 clamped between 1 and 2
+	assert 3 clamped between 1 and 2 is 2 with error "(single ints) min < max < value", expected 2, got 3 clamped between 1 and 2
+	assert 3 clamped between 2 and 1 is 2 with error "(single ints) max < min < value", expected 2, got 3 clamped between 2 and 1
```

Comment:
```suggestion
	assert 1 clamped between 0 and 2 is 1 with error "(single ints) min < value < max"
	assert 1 clamped between 1 and 2 is 1 with error "(single ints) min = value < max"
	assert 2 clamped between 1 and 2 is 2 with error "(single ints) min < value = max"
	assert 0 clamped between 1 and 2 is 1 with error "(single ints) value < min < max"
	assert 3 clamped between 1 and 2 is 2 with error "(single ints) min < max < value"
	assert 3 clamped between 2 and 1 is 2 with error "(single ints) max < min < value"
```

the expected and got values will be auto-filled for you if you use `x is y` as your condition!
this goes for all the others too

Reply:
oh